addons/website/data/website_data.xml

```diff
@@ -17,12 +17,16 @@
     <!-- Template for home and contactus -->
     <template id="website.homepage" name="Home">
         <t t-call="website.layout">
+            <t t-set="homepage_description">This is the homepage page of </t>
+            <t t-set="website_meta_description" t-value="homepage_description + request.env['website'].get_current_website().name"/>
```

Comment:
`"This is the homepage page of "` is not a full sentence, it cannot be translated. Maybe some devs allow it but at least they would probably not make the final space part of the translation? I probably doesn't even work: your .pot does not have the final space, so I guess even in french this would lead to "Ceci est la page d'accueil deSite 1"... You have to make it so the replacement part is part of the translation. Something like `This is the homepage page of %s` then do `homepage_description % request.env['website'].get_current_website().name`. But it's also better in general to avoid that. Maybe it would be easier here to just say "Homepage of the website".

However, in this case, I still want you to fill in the `website_meta_description` ***fields*** of the default pages... not change the arch :/ Probably in python as well since when I create a new website, I guess there is python code bootstrapping those default pages?

Actually I also just tested this... and it does not seem to work at all. I am doing something wrong? Hard to believe you are sending me something that still does not work... and that just seem like it has not been tested at all. With this diff, what I have is the generic default page having this change but not the real website ones:
![image](https://github.com/odoo/odoo/assets/10338094/8f0276cd-ce5c-4ba8-a13e-f02e9509cab4)
(here the two specific homepages act as before, while the generic one which is always unused contains your diff). What am I missing?